python/ray/data/examples/data/iris.tsv

```diff
@@ -0,0 +1,150 @@
+5.1	3.5	1.4	0.2	setosa
```

Comment:
We also need to define the schema at the first line:

```
sepal.length sepal.width petal.length petal.width variety
```

Reply:
Thanks, I fixed that now :)